content/applications/finance/accounting/payments/internal_transfers.rst

```diff
@@ -2,45 +2,40 @@
 Internal transfers
 ==================
 
-In Odoo, internal money transfers can be made in a few clicks. You need at least either two bank
-accounts, two cash journals, or one bank account and one cash journal.
+Internal money transfers can be handled in Odoo. At least two bank accounts are needed to make
+internal transfers.
+
+.. seealso::
+   :doc:`How to add an additional bank account <../bank>`
 
 Configuration
 =============
 
 An internal transfer account is automatically created on your database based on your company's
 localization and depending on your country’s legislation. If needed, the default :guilabel:`Internal
-transfer account` can be modified by going to :menuselection:`Accounting app --> Configuration -->
-Settings` and then under the :guilabel:`Default Accounts` section.
-
-.. note::
-   At least two bank accounts are needed to make internal transfers. Refer to
-   :doc:`Bank and cash accounts section <../bank>` to see how to add an additional bank
-   account to your database.
+transfer account` can be modified by going to :menuselection:`Accounting --> Configuration -->
+Settings --> Default Accounts section`.
 
 Register an internal transfer from one bank to another
 ======================================================
 
-Let's say you have two bank accounts registered on your database and you want to transfer 1,000 USD
-from Bank A to Bank B.
-
 Log an internal transfer
 ------------------------
 
-From the Accounting Dashboard, click on the drop-down selection button (:guilabel:`⋮`) on one of
-your banks. In the :guilabel:`New` column click on :guilabel:`Internal Transfer` and enter the
-information related to the transfer.
-
-.. image:: internal_transfers/internal_transfer.png
-   :align: center
-   :alt: Fill in the information related to your internal transfer
+If you want to transfer 1000 USD from Bank A to Bank B, from the Accounting Dashboard, click on the
+drop-down selection button (:guilabel:`⋮`) on one of your banks, then click on :guilabel:`Payments`.
+Each payment can be marked as :guilabel:`Internal Transfer`, however, a :guilabel:`Destination Journal`
+must be selected in order to :guilabel:`Confirm` the internal transfer.
```

Comment:
I would rephrase this to a) remove the example part of the explanation as i don't feel it's necessary b) phrase the second part as more of a procedure:

```suggestion
If you want to transfer money from one bank to another, access the Accounting Dashboard, click the drop-down selection button (:guilabel:`⋮`) on the bank from which you want to make the transfer, then click guilabel:`Payments`. Select or create a payment, tick the :guilabel:`Internal Transfer` checkbox, and select a :guilabel:`Destination Journal` before you :guilabel:`Confirm` the internal transfer.
```